src/Solver.cpp

```diff
@@ -725,6 +740,14 @@ void Solver::initializeFSRs() {
   _num_groups = _geometry->getNumEnergyGroups();
   _num_materials = _geometry->getNumMaterials();
 
+#ifdef NGROUPS
+  if (_geometry->getNumEnergyGroups() != _num_groups)
+    log_printf(ERROR, "OpenMOC has been compiled for NGROUPS groups, and the "
```

Comment:
I don't think a macro is valid in a string. So please change NGROUPS to %d.

Reply:
done